spec/controllers/ems_common_controller_spec.rb

```diff
@@ -158,3 +158,38 @@
     end
   end
 end
+
+describe EmsInfraController do
+  context "button pressed='miq_template_clone'" do
+    describe "#button" do
+      before do
+        EvmSpecHelper.seed_specific_product_features("ems_infra_new")
+        feature = MiqProductFeature.find_all_by_identifier(["ems_infra_new"])
+        test_user_role = FactoryGirl.create(:miq_user_role,
+                                            :name                 => "test_user_role",
+                                            :miq_product_features => feature)
+        test_user_group = FactoryGirl.create(:miq_group, :miq_user_role => test_user_role)
+        user = FactoryGirl.create(:user, :name => 'test_user', :miq_groups => [test_user_group])
+
+        allow(user).to receive(:server_timezone).and_return("UTC")
+        described_class.any_instance.stub(:set_user_time_zone)
+        controller.stub(:check_privileges).and_return(true)
+        controller.stub(:assert_privileges).and_return(true)
+        login_as user
+      end
+
+      it "returns and avoids a double render error if the flash msg says template cannot be cloned" do
```

Comment:
@martinpovolny Ok, Noted.  Changed the spec description per your recommendation.